apps/storybook/stories/FeaturedQuoteBlock.stories.tsx

```diff
@@ -1,26 +1,23 @@
+import FeaturedQuoteBlock from '@evans/pages/friend/FeaturedQuoteBlock';
```

Comment:
here's an example of one of the prettier imports, without the `../../next-evans/blah/blah`

Reply:
Oh cool, that's nice